redisAgent/config.ini

```diff
@@ -0,0 +1,17 @@
+[user_info]
+username =
+license_key =
+
+[sentinels]
+# it should be [(host, port), ...]
+startup_nodes =
+
+[sysmgr-conf]
```

Comment:
please unify the naming convention as the previous user_info you used the underscore